### DISCUSSION TITLE
WORKING VERSION 2024

### DISCUSSION THREAD

**Discussion by appleburger123** [victornpb/undiscord#588] [1. Questions & Answers] [answered]
# **NOT MY CODE** (The changes are made by @ttuleyb)

## Please upvote if you find this version to be working!

It's kind of slow and you might run into a bunch of error messages and a the code might leave a few messages undetected, but it does the job. The main problem that it solved for me was the empty page problem where it only deletes around 30 messages (believed to be due to a Discord update in 2023). Follow the same steps as any other versions of Undiscord.

I recommend setting search delay to >2000 and delete delay to >500; I deleted over 50k messages with this preset (the video's before I ran into issues).

https://pastebin.com/KibdNwfP

https://github.com/victornpb/undiscord/assets/73975794/fe34bbce-e41e-4b56-ae97-0985fcbbb723

<img width="812" alt="Screenshot 2024-01-29 at 7 01 59 PM" src="https://github.com/victornpb/undiscord/assets/73975794/6146032c-3594-4840-8d03-4d0b133d1268">

**Comment by ttuleyb** [comment, marked as answer]
Yeah I made this a while ago, quickly hacked together a script that simply skips a page if there are no messages in it.

**Reply by jake983394** [reply]
do u have a fix for the problem "Error deleting message, API responded with status 403!	{"message":"Missing Permissions","code":50013}" ? please

**Reply by abdallasamir04** [reply]
yoo 
Try FirFox it's working with me rn 
Download the script 
Create a Script
Copy-Paste Ctrl+s 
refresh Discord Win+Shift+R
Boooooom 
It's working ...

**Comment by Mspy1** [comment]
It works.

**Comment by ClitcherCode** [comment]
Hi. I have an error with 50083 code. Here you can see:
"Error deleting message, API responded with status 400!	{"message":"Ветка заархивирована","code":50083}".
If someont knows what is going on, please, give me to know. Thank you!

**Reply by jake983394** [reply]
did u find a fix for this? im having the same issue

**Comment by Coochiee** [comment]
How do i use this? please help me

**Comment by TheCellMaster** [comment]
it works!
